src/shared/ui/tooltip/tooltip.stories.tsx

```
@@ -1,330 +1,317 @@
-                </Tooltip>
-            </div>
+  args: {
+    children: null,
+  },
+  render: () => (
+    <>
+      {/* Top */}
+      <div className="flex flex-col items-center gap-4">
+        <Tooltip position="top">
+          <Tooltip.Trigger>
+            <button className="rounded-lg bg-gray-700 px-6 py-3 text-white">
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)